src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -283,30 +213,30 @@ static void readConfig(const char *filename)
                 /* chop \n characters */
                 *cp = '\0';
             }
-            log_debug("read config line %u: \"%s\".\n", lineCount, line);
+            debugs(DEBUG_SECTION, 3, "read config line " << lineCount << ": \"" << line << '\"');
             if ((username = strtok(line, "\t ")) != NULL) {
 
                 /* get the time budget */
                 if ((budget = strtok(nullptr, "/")) == NULL) {
-                    fprintf(stderr, "ERROR: missing 'budget' field on line %u of '%s'.\n", lineCount, filename);
+                    std::cerr << "ERROR: missing 'budget' field on line " << lineCount << " of '" << filename << "'.\n";
                     continue;
                 }
                 if ((period = strtok(nullptr, "/")) == NULL) {
-                    fprintf(stderr, "ERROR: missing 'period' field on line %u of '%s'.\n", lineCount, filename);
+                    std::cerr << "ERROR: missing 'period' field on line " << lineCount << " of '" << filename << "'.\n";
```

Comment:
It feels like this should be converted to a DBG_IMPORTANT debugs() as well. Same for the std::cerr a few lines earlier.

Reply:
yes. done